spring-cloud-kubernetes-fabric8-config/src/test/java/org/springframework/cloud/kubernetes/fabric8/config/NamedConfigMapContextToSourceDataProviderTests.java

```diff
@@ -340,4 +345,55 @@ void testCorrectNameWithProfile() {
 		Assertions.assertEquals(sourceData.sourceData(), Collections.singletonMap("key", "value"));
 	}
 
+	/**
```

Comment:
a test that proves that log statements are on par with caching, first request is from k8s, subsequent ones are from cache